template-sar.yml

```diff
@@ -1,17 +1,17 @@
-AWSTemplateFormatVersion: '2010-09-09'
-Transform: 'AWS::Serverless-2016-10-31'
-Description: 'Create pipeline-dashboard application.'
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: "AWS::Serverless-2016-10-31"
+Description: "Create pipeline-dashboard application."
 Parameters:
   PipelinePattern:
-    Description: 'The pattern of pipeline names to allow access to describe.  Recommended value: *'
+    Description: "The pattern of pipeline names to allow access to describe.  Recommended value: *"
     Type: String
 Resources:
   PipelineDashboardEventHandler:
-    Type: 'AWS::Serverless::Function'
+    Type: "AWS::Serverless::Function"
     Properties:
       Description: Create CloudWatch metrics from AWS CodePipeline events
       Handler: index.handlePipelineEvent
-      Runtime: nodejs8.10
+      Runtime: nodejs12.x
       CodeUri: s3://mr-pipeline-dashboard-sar-ap-southeast-2/17e962257e836f209df71b0d43673b70
```

Comment:
If the previous code will work fine in the new environment, then this is fine.  Otherwise this file should be reverted until rebuilt - then separate PR / commit is required.